pvpn-cli/killswitch.conf

```diff
@@ -0,0 +1,33 @@
+#!/bin/env net -f
+
+table inet filter {
+
+    chain input {
+        type filter hook input priority 0; policy drop;
+        # Allow loopback
+        iif lo accept
+
+        # Allow new, established, and related traffic
+        ct state new,established,related accept
```

Comment:
Why do we want to allow incoming connections?

Reply:
The new connections are incoming and are to initially connect with privoxy; otherwise no incoming [connection](https://wiki.nftables.org/wiki-nftables/index.php/Matching_connection_tracking_stateful_metainformation#ct_state_-_conntrack_state) can be made except within proxy itself.

Reply:
I understand what this does, but not clear on why we would and to allow _incoming_ connections at all. The connection to setup the VPN is outgoing, as is all connections using the VPN, right?

In fact, shouldn't the `new` on this line be added to the corresponding `output` rule? Otherwise no new connection can be made _from_ this container.

Reply:
Yes, outgoing connections can already be made as proton0 allows it, but to connect to privoxy as a proxy incoming connection of some form has to be allowed; otherwise, it's not a proxy, but a direct VPN connection from within. I recommend testing your edge cases and letting me know.

Reply:
Right. That makes sense, but it still seems odd that _all_ incoming connections are allowed, and no outgoing ones. I'll test it out a bit.